src/pages/ahnurim/Login/Login.scss

```
@@ -1 +1,69 @@
-@import '../variables.scss';
+body {
+  background-color: #fafafa;
+  display: flex;
+  justify-content: center;
+  padding: 15px;
+  align-items: center;
+}
+
+.login {
+  .boxlayout {
+    display: flex;
+    flex-direction: column;
+    border: solid 0.3px;
+    background-color: white;
+    /* width: 450px;*/
+    height: 600px;
+    padding: 60px;
+  }
+  h1 {
+    font-family: 'Lobster';
+    font-weight: 1px;
+    font-size: 55px;
+    display: flex;
+    justify-content: center;
+  }
+
+  .box {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 10px;
+  }
+  input {
+    display: flex;
+    justify-content: center;
+    width: 300px;
+    height: 30px;
+    color: #999999;
+    margin: 4px;
+    border: solid;
+    color: #999999;
+    border-radius: 5px;
+    background-color: #fafafa;
+    font-size: 12px;
+  }
+  button {
+    display: flex;
+    justify-content: center;
+    width: 308px;
+    /* height: 30px; */
+    padding: 10px 0px;
+    background-color: #c1dffd;
+    text-align: center;
+    /* border: 2px; solid; border-radius: 2px; */
+    color: white;
+    border: solid;
+    border-radius: 5px;
+    &.Active {
+      background-color: rgb(0, 75, 179);
+    }
+  }
```

```suggestion
  input {
    display: flex;
    justify-content: center;
    width: 300px;
    height: 30px;
    color: #999999;
    margin: 4px;
    border: solid;
    color: #999999;
    border-radius: 5px;
    background-color: #fafafa;
    font-size: 12px;
  }
  
  button {
    display: flex;
    justify-content: center;
    width: 308px;
    /* height: 30px; */
    padding: 10px 0px;
    background-color: #c1dffd;
    text-align: center;
    /* border: 2px; solid; border-radius: 2px; */
    color: white;
    border: solid;
    border-radius: 5px;
    
    &.Active {
      background-color: rgb(0, 75, 179);
    }
  }
```
태그에 직접 스타일을 적용하는것은 최대한 지양해주세요!
그리고 코드의 가독성을 위해서 선택자끼리는 한칸씩 띄어서 작성해주세요